CHANGELOG.md

```diff
@@ -58,12 +58,14 @@
   - synchronous contract calls
   - fallback entrypoints
   - increased smart contract module size limit, 512kB
+- Node can now be stopped during out of band catchup by using signals like SIGINT.
```

Comment:
Specifically SIGINT and SIGTERM is what is handled explicitly. The other signals have system defined behaviour.

Reply:
I suggest you are more precise here.

Reply:
done 👍